homework/linux.txt

```diff
@@ -0,0 +1,141 @@
+   26  cat selected.txt 
+   27  history
+arturszczygiel@arturszczygiel-VirtualBox:~/test/linux$ history | tail -28 > commands.txt
+arturszczygiel@arturszczygiel-VirtualBox:~/test/linux$ cat selected.txt commands.txt 
+199 processes.txt
+processes.txt:87:message+     576  0.0  0.1   8892  5896 ?        Ss   08:09   0:00 /usr/bin/dbus-daemon --system --address=systemd: --nofork --nopidfile --systemd-activation --syslog-only
+processes.txt:89:root         595  0.0  0.5  41860 20196 ?        Ss   08:09   0:00 /usr/bin/python3 /usr/bin/networkd-dispatcher --run-startup-triggers
+processes.txt:100:root         701  0.0  0.5 120556 22740 ?        Ssl  08:09   0:00 /usr/bin/python3 /usr/share/unattended-upgrades/unattended-upgrade-shutdown --wait-for-signal
+processes.txt:106:whoopsie     758  0.0  0.3 327256 15940 ?        Ssl  08:09   0:00 /usr/bin/whoopsie -f
+processes.txt:110:artursz+     813  0.0  0.4 2722188 20024 ?       S<sl 08:09   0:01 /usr/bin/pulseaudio --daemonize=no --log-target=journal
+processes.txt:112:artursz+     819  0.0  0.1 242716  7728 ?        SLl  08:09   0:00 /usr/bin/gnome-keyring-daemon --daemonize --login
+processes.txt:113:artursz+     823  0.0  0.1 166548  6600 tty2     Ssl+ 08:09   0:00 /usr/lib/gdm3/gdm-x-session --run-script env GNOME_SHELL_SESSION_MODE=ubuntu /usr/bin/gnome-session --systemd --session=ubuntu
+processes.txt:115:artursz+     831  0.0  0.2  11156  8252 ?        Ss   08:09   0:00 /usr/bin/dbus-daemon --session --address=systemd: --nofork --nopidfile --systemd-activation --syslog-only
+processes.txt:128:artursz+    1035  0.0  0.0   6040   452 ?        Ss   08:09   0:00 /usr/bin/ssh-agent /usr/bin/im-launch env GNOME_SHELL_SESSION_MODE=ubuntu /usr/bin/gnome-session --systemd --session=ubuntu
+processes.txt:130:artursz+    1067  0.0  0.1   7244  4204 ?        S    08:09   0:00 /usr/bin/dbus-daemon --config-file=/usr/share/defaults/at-spi2/accessibility.conf --nofork --print-address 3
+processes.txt:133:artursz+    1095  3.0 10.2 3781036 413472 ?      Ssl  08:09   5:22 /usr/bin/gnome-shell
+processes.txt:147:artursz+    1225  0.0  0.6 2601292 26584 ?       Sl   08:10   0:00 /usr/bin/gjs /usr/share/gnome-shell/org.gnome.Shell.Notifications
+processes.txt:172:artursz+    1522  0.0  1.9 1008600 79316 ?       Sl   08:12   0:10 /usr/bin/nautilus --gapplication-service
+processes.txt:195:artursz+   14952  0.1  1.4 820808 56664 ?        Sl   11:02   0:00 /usr/bin/gedit --gapplication-service
+    1  clear
+    2  pwd
+    3  mkdir linux
+    4  ls
+    5  cd linux
+    6  touch processes.txt
+    7  ls
+    8  ps aux > processes.txt
+    9  ls
+   10  less processes.txt 
+   11  cp processes.txt copy.txt
+   12  ls
+   13  mkdir backup
+   14  mv copy.txt backup/
+   15  ls
+   16  cd backup/
+   17  ls
+   18  cd ..
+   19  ls
+   20  wc -l processes.txt 
+   21  wc -l processes.txt > selected.txt
+   22  ls
+   23  less selected.txt 
+   24  grep -n /bin *
+   25  grep -n /bin * >> selected.txt 

```

Comment:
You should grep in processes.txt file, not in every file in current directory

Reply:
Thanks for advice.